app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -50,7 +50,7 @@
     private BluetoothAdapter bluetoothAdapter;
     private IBluetoothHandler bluetoothHandler;
     private Set<BluetoothDevice> pairedDevices;
-    private boolean isMaster = false;
+    private boolean isMaster;
```

Comment:
Checkstyle can only enforce, just like Intellij, only that variables are not initiallized to their default values, see http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/coding/ExplicitInitializationCheck.html